plugins/wrappers/backendwrapper/backendwrapper.go

```diff
@@ -173,8 +173,8 @@ func (b *Backend) TxPoolContent() (map[core.Address][][]byte, map[core.Address][
 func (b *Backend) BloomStatus() (uint64, uint64) {
 	return b.b.BloomStatus()
 }
-func (b *Backend) GetLogs(ctx context.Context, blockHash core.Hash) ([][]byte, error) {
-	logs, err := b.b.GetLogs(ctx, common.Hash(blockHash))
+func (b *Backend) GetLogs(ctx context.Context, blockHash core.Hash, number uint64) ([][]byte, error) {
```

Comment:
Instead of changing this signature (in turn having to change PluGeth-utils and any plugins that rely on this) let's leave this signature the same, but then add:

```
header := b.b.GetHeaderByNumber(ctx, common.Hash(blockHash))
```

then change the GetLogs call to 

```
	logs, err := b.b.GetLogs(ctx, common.Hash(blockHash), header.Number)
```